lib/chef/knife/bootstrap/bootstrap_options.rb

```diff
@@ -90,6 +90,13 @@ def self.included(includer)
                                   It specifies the frequency (in minutes) at which the chef-service runs.
                                   Pass 0 if you don't want the chef-service to be installed on the target machine."
 
+            option :daemon,
+              :long => "--daemon DAEMON",
+              :description => "Optional, Provide this option when --bootstrap-protocol is set to 'cloud-api'. Works only for Windows node.
```

Comment:
"Optional. Configures the Chef client service for unattended execution. Requires --bootstrap-protocol to be 'cloud-api' and the node platform to be Windows.